src/blocks/separator/styling.js

```diff
@@ -5,6 +5,30 @@
 import generateCSS from '@Controls/generateCSS';
 import generateCSSUnit from '@Controls/generateCSSUnit';
 import { getFallbackNumber } from '@Controls/getAttributeFallback';
+const alignmentCSS = ( align ) => {
+	let alignCSS = {};
+	switch ( align ) {
+		case 'left':
+					alignCSS = {
+						'margin-left': 0,
+						'margin-right': 'auto',
+					};
+					break;
+		case 'center':
+					alignCSS = {
+						'margin-left': 'auto',
+						'margin-right': 'auto',
+					};
+					break;
+		case 'right':
+					alignCSS = {
+						'margin-right': 0,
+						'margin-left': 'auto',
+					};
+					break;
+	}
```

Comment:
@AbhijitNage123 - Should we indent it like this?
( Same for PHP )

```suggestion
	switch ( align ) {
		case 'left':
			alignCSS = {
				'margin-left': 0,
				'margin-right': 'auto',
			};
			break;
		case 'center':
			alignCSS = {
				'margin-left': 'auto',
				'margin-right': 'auto',
			};
			break;
		case 'right':
			alignCSS = {
				'margin-right': 0,
				'margin-left': 'auto',
			};
			break;
	}
```

Reply:
Done